piet-direct2d/src/d2d.rs

```diff
@@ -408,6 +413,21 @@ impl DeviceContext {
         unsafe { self.0.SetTarget(target.inner.as_raw() as *mut ID2D1Image) }
     }
 
+    /// Get the target for the device context.
+    pub fn get_target(&mut self, target: &Image) {
+        unsafe { self.0.GetTarget(&mut target.inner.as_raw()) }
+    }
+
+    pub fn get_target_wat(&mut self) -> Image {
+        unsafe {
+            let mut foo: *mut ID2D1Image = std::ptr::null_mut();
+            self.0.GetTarget(&mut foo);
+            Image {
+                inner: ComPtr::from_raw(foo),
+            }
+        }
+    }
```

Comment:
@x3ro do you have any context for these two methods? I don't see it used anywhere and I'm not sure what it's trying to do.

Reply:
Oh lord. If they're not used anymore they can definitely be removed. I had some problems with the windows APIs reacting differently to different kinds of "null-ish" pointers, so this might've been me experimenting and then forgetting to remove them 😱